src/AirshipBindings.NETStandard/AirshipBindings.NETStandard.iOS/Airship.cs

```diff
@@ -244,6 +244,34 @@ public void DisplayMessage(string messageId)
             UAMessageCenter.Shared().DisplayMessage(messageId);
         }
 
+        public void MarkMessageRead(string messageId)
+        {
+            var messages = UAMessageCenter.Shared().MessageList.Messages;
+            foreach (var message in messages)
+            {
+                if (message.MessageID == messageId)
+                {
+                    message.MarkMessageRead(null);
+                    Console.WriteLine("Message read !");
+                }
+            }
+        }
+
+        public void DeleteMessage(string messageId)
+        {
+            var toDelete = new UAInboxMessage[1];
+            var messages = UAMessageCenter.Shared().MessageList.Messages;
```

Comment:
Should be able to get the message with messageForID method

Reply:
Looks like the method messageForID is not exposed, I found it in ApiDefinitions.cs (cf this screenshot), but couldn't retrieve it in the code, I don't know why
![image](https://user-images.githubusercontent.com/15062214/92403328-86792800-f131-11ea-9bcc-53f10f062373.png)

Reply:
`var message = UAMessageCenter.Shared().MessageList.Message(messageId);`